src/CoreExtensions/ClassDefinitions/DynamicPermissionResource.php

```diff
@@ -28,9 +28,7 @@ class DynamicPermissionResource extends Data implements Data\ResourcePersistence
     use DataObject\Traits\SimpleComparisonTrait;
 
     /**
-     * Static type of this element
-     *
-     * @var string
+     * @deprecated Will be removed in frontend-permission-toolkit 2, use getFieldType() instead.
```

Comment:
```suggestion
     * @deprecated Will be removed in frontend-permission-toolkit 3, use getFieldType() instead.
```
since 2 is already released https://github.com/pimcore/frontend-permission-toolkit/releases/tag/v2.0.0

Reply:
@kingjia90 Changed it: https://github.com/pimcore/frontend-permission-toolkit/pull/59/commits/ae09ae18b74aee914618d7bbaec77165adfa042c